Shift3/boilerplate-client-react
# Architecture refactor

## Changes
1. Removed `index.tsx` files in `src/common/components`. 
2. Renamed `index.tsx` with component named files.
3. Updated all imports.
4. Moved styled-components into various `/styles` files including `form` `page` and `utilities`.
5. Added a `card.tsx` in `src/common/styles`

## Purpose
To remove an additional directory that is unecessary.

## Approach
Removed all index.tsx files and replaced with component named files, unless the component had multiple files related to it, then a directory for that component would remain.

## Learning
The architecture needed some clean up. Removing and renaming these files takes away one extra layer to help simplify finding components and navigating the app.

## Pre-Testing TODOs
Run the `node server`

## Testing
1. Pull in the changes to your local copy of this branch and run `yarn start`.
2. Make sure the app renders and you can navigate without errors.
